### DISCUSSION TITLE
Custom stack usage in pack command

### DISCUSSION THREAD

**Discussion by hariohmprasath** [paketo-buildpacks/feedback#186] [Q&A]
_Have a question related to custom stacks:_
I created a custom stack using `jam` command, is there a way to specify the stack that should get used part of the `pack build` execution, instead of forking the entire builder and just replacing the `[stack]` section in `builder.toml` file. Thanks

**Comment by dmikusa** [comment]
No. You can change just the run image with an argument when you `pack build`. You can't change the build image though, unless you create a custom builder.
